### DISCUSSION TITLE
Reminders for an entire CalDav synced calendar?

### DISCUSSION THREAD

**Discussion by shlmshdi** [FossifyOrg/General-Discussion#475] [Q&A] [answered]
Hello there, im sorry if i can't explain myself as i should, english is not my native language. having said that, i am starting to use this great calendar and everything has been fantastic so far, i do have one question though, in google calendar, syncing an ical calendar makes something like a label for identifying it, so does fossify cal; the thing that im trying to do here is that in google calendar you can adjust it to make reminders x time before any event coming from said synced calendar, but i cant find the option to do that in fossify cal, is this even possible or has this feature not been added yet? if so i'd gladly make a feature request for them, but i really need this so i can fully transition from google cal to fossify cal. Thanks to everyone in advance <3

**Comment by Aga-C** [comment, marked as answer]
As I understand you correctly, you'd like to set the same reminders for all events of a given type? If so, we don't have such feature right now, you can raise a request in [Issues](https://github.com/FossifyOrg/Calendar/issues/new?assignees=&labels=feature+request%2Cneeds+triage&projects=&template=feature_request.yml).

**Reply by shlmshdi** [reply]
ahh ok, thank you :)
